lms/djangoapps/verify_student/houston_encrypt.py

```diff
@@ -0,0 +1,125 @@
+
+import json
+import logging
+import requests
+import boto3
+
+from datetime import datetime, timedelta
+from email.utils import formatdate
+from django.conf import settings
+from django.core.urlresolvers import reverse
+log = logging.getLogger(__name__)
+import pdb
+
+from lms.djangoapps.verify_student.ssencrypt import (
+    generate_signed_message
+)
+
+def create_houston_request(receipt_id, photo_id_key, copy_id_photo_from=None):
+    """
+    Construct the HTTP request to the photo verification service.
+    Keyword Arguments:
+        copy_id_photo_from (SoftwareSecurePhotoVerification): If provided, re-send the ID photo
+            data from this attempt.  This is used for reverification, in which new face photos
+            are sent with previously-submitted ID photos.
+    Returns:
+        tuple of (header, body), where both `header` and `body` are dictionaries.
+    """
+    scheme = "https" if settings.HTTPS == "on" else "http"
+    callback_url = "{}://{}{}".format(
+        scheme, settings.SITE_NAME, reverse('verify_student_results_callback')
```

Comment:
does not look like microsites are being respected here, so we must just make super duper sure this is not breaking anything. might just be fine

Reply:
This is also the default behaviour of Software secure so I tried to keep the (`create_request` and `create_houston_request`) functions as similar as possible.

Reply:
So I've changed it to use configurations helpers for the site name.